network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -10,15 +10,18 @@
 {% block subcontent %}
 <div class="tw-container">
   <div class="tw-row">
-    <div class="
+
+      <div class="
       {% block content_wrapper_class %}tw-py-5 medium:tw-py-7{% endblock %}
       {% if page.specific.narrowed_page_content %}
         tw-w-8/12 tw-mx-auto tw-px-4
+      {% elif page.aside %}
+        medium:tw-w-3/4 tw-px-4
       {% else %}
         tw-w-full tw-px-4
       {% endif %}"
```

Comment:
I think all that is missing is that we need to define a base width (before the medium breakpoint is reached) for the case with the aside. We don't want to use the same one that comes for the narrow page, because that is the same as we use after the medium breakpoint when we have an aside (but the narrow page also centers).

```suggestion
      {% if page.specific.narrowed_page_content and not page.aside %}
        tw-w-3/4 tw-mx-auto tw-px-4
      {% elif page.aside %}
        tw-w-full medium:tw-w-3/4 tw-px-4
      {% else %}
        tw-w-full tw-px-4
      {% endif %}"
```

Reply:
https://user-images.githubusercontent.com/24797493/194147499-1dc78be7-5907-431b-b711-4a8e0e88f76f.mov

The fact that the button is not showing right should not have anything to do with your changes. That feels like an issue with the link button block.

Reply:
Thanks! added in https://github.com/mozilla/foundation.mozilla.org/pull/9434/commits/281351ec5b4033905dffb77c146b8cb77e08472e - I'm using the `large` breakpoint as I think that's a lof of content to squeeze in on smaller screens